ts/src/main/scala/ch/epfl/ts/traders/SimpleFXTrader.scala

```diff
@@ -43,39 +49,38 @@ class SimpleFXTrader(val uid: Long, val shortPeriod: Int, val longPeriod : Int,
           currentShort = ma.value
           shortMaCount += 1
         }
-        case`longPeriod` => {
+        case `longPeriod` => {
           println("received a long period")
           previousLong = currentLong
           currentLong = ma.value
-          longMaCount += 1 
+          longMaCount += 1
         }
 
-        case _ => println( "SimpleFXTrader: received unknown message" )
+        case _ => println("SimpleFXTrader: received unknown message")
 
       }
-        if(shortMaCount == longMaCount && priceReady) { //we make sure that we are comparing MA from the same period
-            if (previousShort < previousLong && currentShort >= currentLong) {  //if short goes above long buy
-              send(MarketBidOrder(oid, uid,System.currentTimeMillis(), USD, CHF, volume, bidPrice))
-              println("simple trader : buying")
-              oid += 1
-            }
-            
-            else if(previousShort > previousLong && currentShort <= currentLong) { //if short goes below long sell
-              send(MarketAskOrder(oid, uid, System.currentTimeMillis(), USD, CHF, volume, askPrice))
-              println("simple trader : selling")
-              oid += 1
-            }
-          }   
+      if (shortMaCount == longMaCount && priceReady) { //we make sure that we are comparing MA from the same period
+        if (previousShort < previousLong && currentShort >= currentLong) { //if short goes above long buy
+          send(MarketBidOrder(oid, uid, System.currentTimeMillis(), USD, CHF, volume, bidPrice))
+          println("simple trader : buying")
+          oid += 1
+        } else if (previousShort > previousLong && currentShort <= currentLong) { //if short goes below long sell
+          send(MarketAskOrder(oid, uid, System.currentTimeMillis(), USD, CHF, volume, askPrice))
```

Comment:
Currencies being traded should no longer be hardcoded, since `symbol` is now a parameter.